src/js/utils/html.js

```diff
@@ -0,0 +1,19 @@
+import { getBaseApiUrl } from "../api";
+/*
+  Takes an HTML string and replaces each 'src' attribute in each 'img' tag with
+  the API url prepended.
+
+  E.g. /path/to/img.jpg -> http://api.com/path/to/img.jpg
+ */
+export function parseImgTags(html) {
+  let parsedHTML = document.createElement("div");
+  parsedHTML.innerHTML = html;
+  let imgTags = parsedHTML.querySelectorAll("img");
```

Comment:
These can both be `const` as you are not reassigning

Reply:
thanks, changed :)